ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__hypershift-4.13-lp-rosa-hypershift.yaml

```diff
@@ -0,0 +1,213 @@
+  to: serverless-operator-src
+- context_dir: .
+  dockerfile_path: serving/ingress/Dockerfile
+  from: base
+  inputs:
+    openshift_release_golang-1.19:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.19
+  to: knative-openshift-ingress
+- context_dir: .
+  dockerfile_path: knative-operator/Dockerfile
+  from: base
+  inputs:
+    openshift_release_golang-1.19:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.19
+  to: knative-operator
+- context_dir: .
+  dockerfile_path: openshift-knative-operator/Dockerfile
+  from: base
+  inputs:
+    openshift_release_golang-1.19:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.19
+  to: openshift-knative-operator
+- context_dir: .
+  dockerfile_path: serving/metadata-webhook/Dockerfile
+  from: base
+  inputs:
+    openshift_release_golang-1.19:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.19
+  to: metadata-webhook
+- context_dir: olm-catalog/serverless-operator
+  from: base
+  to: serverless-bundle
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-hypershift-413-continuous
+  cron: 0 10 * * 1
+  remote_api: true
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPTOCPCI"},
+          {"step": "cucushift-hypershift-*", "failure_type": "all", "classification": "Cluster Health Check", "jira_project": "LPTOCPCI"},
+          {"step": "aws-*", "failure_type": "all", "classification": "Infrastructure AWS", "jira_project": "LPTOCPCI"},
+          {"step": "gather-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI"},
+          {"step": "osd-*", "failure_type": "all", "classification": "User Setup", "jira_project": "LPTOCPCI"},
+          {"step": "rosa-", "failure_type": "all", "classification":  "Cluster Creation", "jira_project": "LPTOCPCI"},
+          {"step": "knative-eventing-kafka-broker-e2e", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "SRVCOM", "jira_epic": "LPTOCPCI-290"},
+          {"step": "knative-serving-eventing-e2e", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "SRVCOM", "jira_epic": "LPTOCPCI-290"},
+          {"step": "operator-e2e", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "SRVCOM", "jira_epic": "LPTOCPCI-290"}
+        ]
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPTOCPCI
+      FIREWATCH_JIRA_SERVER: https://issues.redhat.com
+      HYPERSHIFT_BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      HYPERSHIFT_NODE_COUNT: "4"
```

Comment:
It looks like you've added this var to the defaults of the [rosa-sts-cluster-provision ref](https://steps.ci.openshift.org/reference/rosa-sts-cluster-provision) but the script that the ref runs does not make use of this var anywhere.

What is the reason for adding:
HYPERSHIFT_NODE_COUNT
HYPERSHIFT_BASE_DOMAIN
BASE_DOMAIN

I don't believe any of these are needed.

Reply:
@ascerra These 3 environmental variables are being used in the Serverless QE team in 4.12(https://github.com/openshift/release/pull/39725). I am using them the same for 4.13, not only to make them consistent but in case they are referenced in the tests.
@mgencur Could you please confirm if these variables are needed?

Reply:
We need to run with at least 4 nodes to make the tests pass. The three variables are required in our case because we're using workflow `cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest`. I'm not sure if they're required in your case. We don't reference them directly from tests so they're only required by the workflow to spin up the right cluster.